### PR TITLE
induction exampleData

### PR DESCRIPTION
**Pull request by romainAA**
fixed a problem with fb modal and added the full set example to the exampleData

### CODE REVIEW

File: ac/ac-induction/src/TestPart/ModalSubmit.jsx [hunk 1]
```diff
@@ -12,116 +12,129 @@ import {
...
-  ][data.indexCurrent].correction || {
-    result: 2,
-    show: 0,
-    propertiesIndex: []
-  };
-  return (
-    <Modal isOpen={data.feedbackOpen} contentLabel="Modal">
-      <div style={{ display: 'flex', flexDirection: 'row' }}>
-        <div>
+class ModalSubmit extends Component<Object> {
```
Comment (houshuang) [inline ac/ac-induction/src/TestPart/ModalSubmit.jsx:15]: Can you move this to componentDidMount? You should not do any DOM related stuff in constructor (only setstate etc)

Comment (houshuang) [reply ac/ac-induction/src/TestPart/ModalSubmit.jsx:15]: Actually you don't need a constructor here anyway

File: ac/ac-induction/src/index.js [hunk 2]
```diff
@@ -29,17 +29,17 @@ const meta = {
     {
```
Comment (houshuang) [inline ac/ac-induction/src/index.js]: can you put the exampleData in a separate file and import it?

**Comment by romainAA** [comment]
@houshuang done

**Pull request merged**